buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+import org.gradle.kotlin.dsl.`kotlin-dsl`
+
+plugins {
+    `kotlin-dsl`
+}
+repositories {
+    jcenter()
```

Comment:
Is this really needed? Isn't enough with the `jcenter()` included on the project gradle?

Reply:
when eliminating these lines, the following error appears when synchronizing "Could not find org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.3.21.
Required by: project :buildSrc"